var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,11 +34,9 @@ class Relion(CMakePackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     url      = "https://github.com/3dem/relion"
 
-    version('2.1-beta-1', git='https://github.com/3dem/relion.git',
-            commit='e7607a869687b636d3c39e0d5b6a9cba930fc3b2')
-    version('2.0.3', git='https://github.com/3dem/relion.git',
-            preferred='true')
-
+    version('2.0.3', git='https://github.com/3dem/relion.git', tag='2.0.3')
+    version('2.1', git='https://github.com/3dem/relion.git', tag='2.1',
+        preferred='true')
```

Comment:
no need for `preferred`, it will be chosen automatically.

Reply:
Because it's the higher number?

Reply:
yes, also please use descending order. That's how we it's done in most packages.

Reply:
Done!